pkgs/development/tools/parsing/tree-sitter/grammars/default.nix

```
@@ -1,125 +1,81 @@
-  tree-sitter-query = lib.importJSON ./tree-sitter-query.json;
-  tree-sitter-r = lib.importJSON ./tree-sitter-r.json;
-  tree-sitter-regex = lib.importJSON ./tree-sitter-regex.json;
-  tree-sitter-rego = lib.importJSON ./tree-sitter-rego.json;
-  tree-sitter-river = lib.importJSON ./tree-sitter-river.json;
-  tree-sitter-rst = lib.importJSON ./tree-sitter-rst.json;
-  tree-sitter-ruby = lib.importJSON ./tree-sitter-ruby.json;
-  tree-sitter-rust = lib.importJSON ./tree-sitter-rust.json;
-  tree-sitter-scala = lib.importJSON ./tree-sitter-scala.json;
-  tree-sitter-scheme = lib.importJSON ./tree-sitter-scheme.json;
-  tree-sitter-scss = lib.importJSON ./tree-sitter-scss.json;
-  tree-sitter-smithy = lib.importJSON ./tree-sitter-smithy.json;
-  tree-sitter-solidity = lib.importJSON ./tree-sitter-solidity.json;
-  tree-sitter-sparql = lib.importJSON ./tree-sitter-sparql.json;
-  tree-sitter-sql = lib.importJSON ./tree-sitter-sql.json;
-  tree-sitter-supercollider = lib.importJSON ./tree-sitter-supercollider.json;
-  tree-sitter-surface = lib.importJSON ./tree-sitter-surface.json;
-  tree-sitter-svelte = lib.importJSON ./tree-sitter-svelte.json;
-  tree-sitter-talon = lib.importJSON ./tree-sitter-talon.json;
-  tree-sitter-templ = lib.importJSON ./tree-sitter-templ.json;
-  tree-sitter-tera = lib.importJSON ./tree-sitter-tera.json;
-  tree-sitter-tiger = lib.importJSON ./tree-sitter-tiger.json;
-  tree-sitter-tlaplus = lib.importJSON ./tree-sitter-tlaplus.json;
-  tree-sitter-toml = lib.importJSON ./tree-sitter-toml.json;
-  tree-sitter-tsq = lib.importJSON ./tree-sitter-tsq.json;
-  tree-sitter-turtle = lib.importJSON ./tree-sitter-turtle.json;
-  tree-sitter-twig = lib.importJSON ./tree-sitter-twig.json;
-  tree-sitter-typescript = lib.importJSON ./tree-sitter-typescript.json;
-  tree-sitter-typst = lib.importJSON ./tree-sitter-typst.json;
-  tree-sitter-uiua = lib.importJSON ./tree-sitter-uiua.json;
-  tree-sitter-verilog = lib.importJSON ./tree-sitter-verilog.json;
-  tree-sitter-vim = lib.importJSON ./tree-sitter-vim.json;
-  tree-sitter-vue = lib.importJSON ./tree-sitter-vue.json;
-  tree-sitter-wgsl = lib.importJSON ./tree-sitter-wgsl.json;
-  tree-sitter-wing = lib.importJSON ./tree-sitter-wing.json;
-  tree-sitter-yaml = lib.importJSON ./tree-sitter-yaml.json;
-  tree-sitter-yang = lib.importJSON ./tree-sitter-yang.json;
-  tree-sitter-zig = lib.importJSON ./tree-sitter-zig.json;
-}
+  lib,
+  fetchFromGitHub,
+  fetchFromGitLab,
+  fetchFromSourcehut,
+  nix-update-script,
+}:
+
+let
+  /**
+    Set of grammar sources. See ./grammar-sources.nix to define a new grammar.
+  */
+  grammar-sources = import ./grammar-sources.nix { inherit lib; };
+
+  /**
+    Attribute to be inserted as the default grammar updater.
+  */
+  updateScript = nix-update-script {
+    extraArgs = [
+      "--override-filename pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources.nix"
+    ];
```

```suggestion
    extraArgs = [
      "--override-filename"
      "pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources.nix"
    ];
```

I had to change this, with the current args I ran into:
```
error:
       … while evaluating attribute 'cargo_deps'
         at «string»:42:3:
           41|   go_modules_old = pkg.go-modules.outputHash or null;
           42|   cargo_deps = pkg.cargoDeps.outputHash or null;
             |   ^
           43|   cargo_vendor_deps = pkg.cargoDeps.vendorStaging.outputHash or null;

       error: attribute '"--override-filename pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources"' missing
       at «string»:8:9:
            7|            (if args ? overlays then { overlays = [ ]; } else {});
            8|   pkg = (pkgs inputs)."--override-filename pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources"."nix";
             |         ^
            9|   sanitizePosition = x: x;
```